Description: Removes privacy breach
 The URL of the icon "Fork me on GitHub" is a privacy breach, and is removed
 by this patch.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-06-01

--- python-itsdangerous-0.24+dfsg1.orig/docs/_themes/flask_small/layout.html
+++ python-itsdangerous-0.24+dfsg1/docs/_themes/flask_small/layout.html
@@ -18,7 +18,7 @@
 {% block relbar2 %}
   {% if theme_github_fork %}
     <a href="http://github.com/{{ theme_github_fork }}"><img style="position: fixed; top: 0; right: 0; border: 0;"
-    src="http://s3.amazonaws.com/github/ribbons/forkme_right_darkblue_121621.png" alt="Fork me on GitHub" /></a>
+    src="broken-link" alt="broken-image-becahse-of-privacy-breach" /></a>
   {% endif %}
 {% endblock %}
 {% block sidebar1 %}{% endblock %}
